README.md

```diff
@@ -0,0 +1,118 @@
+# Telegram Storage
+
+An awesome project to leverage the Telegram Bot API for storing and retrieving files easily.
+
+[Report Bug](https://github.com/1mpossible-code/telegram-storage/issues) · [Request Feature](https://github.com/1mpossible-code/telegram-storage/issues)
+
+## Table of Contents
+
+- [About The Project](#about-the-project)
+- [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+
+## About The Project
+
+![ScreenRecording2024-06-01at8 24 39PM-ezgif com-video-to-gif-converter](https://github.com/1mpossible-code/telegram-storage/assets/109933928/6bf007f4-e813-4562-b2ca-7fc8cef585e6)
+
+Telegram Storage leverages the Telegram Bot API to store and retrieve files easily. It allows users to upload, store, and manage files directly through a Telegram bot, making it convenient to access your files from anywhere.
+
+Here's why this project stands out:
+- **Convenience:** Easily upload and retrieve files through Telegram.
+- **Accessibility:** Access your files from anywhere with Telegram.
+- **Simplicity:** Easy setup and usage instructions.
+
+[back to top](#table-of-contents)
+
+## Built With
+
+This section lists any major frameworks/libraries used to bootstrap the project.
+
+- Python
+- Telegram Bot API
+- Docker
+- Node.js
+
+[back to top](#table-of-contents)
+
+## Getting Started
+
+To get a local copy up and running, follow these simple steps.
+
+### Prerequisites
+
+Ensure you have the following installed:
+- Docker
+- Python
+- Node.js
+- npm
+- pnpm
+
+### Installation
+
+1. **Start Docker containers:**
+    ```bash
+    docker-compose up
+    ```
+
+2. **Create a virtual environment and install Python dependencies:**
+    ```bash
+    python -m venv venv
+    source venv/bin/activate  # On Windows use `venv\Scripts\activate`
+    pip install -r requirements.txt
+    ```
+
+3. **Create a `.env` file:**
```

Comment:
lol, it should be before we run it

Reply:
and do we need it at all? o thought we moved secrets somewhere else. double check that